phpBB/install/phpbbcli.php

```diff
@@ -39,6 +39,9 @@
 // Enable superglobals for cli support
 $phpbb_installer_container->get('request')->enable_super_globals();
 
+// Load compatibility globals to access deprecated globals
+require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
```

Comment:
Confirmed `register_compatibility_globals()` can not be used in the installer CLI because the containers are different, and most of the services in register globals do not exist in the installer's container. So no need to call that method.